server/src/modules/auth/guards/user-auth.guard.ts

```diff
@@ -22,14 +23,24 @@ export class AuthGuard implements CanActivate {
     }
 
     const userAccessToken = gqlContext.headers.authorization.split(' ').pop();
+    try {
```

Comment:
Use refresh token as mutation argument. It's like "password". You don't sent password via headers, do you? :wink: